src/screens/General/Welcome.js

```diff
@@ -0,0 +1,159 @@
+import React, { Component } from "react";
+import {
+	View,
+	SafeAreaView,
+	Text,
+	Image,
+	StyleSheet,
+	StatusBar,
+	TouchableOpacity,
+	Dimensions
+} from "react-native";
+import AppIntroSlider from "react-native-app-intro-slider";
+import { Actions } from "react-native-router-flux";
+
+const { width } = Dimensions.get("screen");
+const data = [
+	{
+		title: "Stay Connected",
+		text: "Stay up-to-date with upcoming GBMs, workshops, social events, networking oppurtunities, and more!",
+		image: require("../../assets/images/welcome1.jpg")
+	},
+	{
+		title: "Get Involved",
+		text: "Subscribe to committees, RSVP to events, vote in club elections, and keep track of your points as you rise through the leaderboard ranks!",
+		image: require("../../assets/images/welcome2.jpg")
+	},
+	{
+		// gif support in build.gradle
+		title: "Welcome to the Familia",
+		text: "We're glad to have you join the SHPE UCF famila!\nWe can't wait to see you succeed alongside us.\n\n Now lets get you all set up. . .",
+		image: require("../../assets/images/welcome3.jpg")
+	}
+];
+
+export class Welcome extends Component {
+switchToLogin = () => {
+	Actions.login();
+}
+
+_renderItem = ({ item }) => {
```

Comment:
* Remove underscore
* Destructure item (title, image, text) in the parameter itself